routes/api.js

```diff
@@ -1,12 +1,60 @@
 const express = require( "express" );
 const router = express.Router();
 
-// /api routes
-router.get( "/", ( req, res ) => {
-  res.sendStatus( 200 );
-} );
-router.get( "/:anything", ( req, res ) => {
-  res.sendStatus( 200 );
-} );
+// todo crud
+router.route( "/todos" )
```

Comment:
Warum hast du hier `/todos` zwei mal?

Reply:
`/todos` (plural) - für alles was mit mehreren todos zu tun hat.
`/todo` - fasst nur ein todo an, Bezug auf `id`s

Reply:
Das ist aber nicht nach REST. In REST hast du `/todos` für alle (`index`) und `/todos/{id}` für einzelne. Schaue dazu unsere Vorlesungsfolien an.